app-scientific/stellarsolver/autobuild/defines

```diff
@@ -0,0 +1,4 @@
+PKGNAME=stellarsolver
+PKGSEC=math
+PKGDEP="gsl wcslib qt-5"
+PKGDES="Star extractor and astrometric solver"
```

Comment:
Stellar data extractor and astrometric solver

Reply:
Done